src/sentry/models/pullrequest.py

```diff
@@ -87,3 +88,19 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_pullrequest_commit"
         unique_together = (("pull_request", "commit"),)
+
+
+@region_silo_only_model
+class PullRequestComment(Model):
+    __include_in_export__ = False
+
+    external_id = BoundedBigIntegerField()
+    pull_request = FlexibleForeignKey("sentry.PullRequest")
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()
+    issues = ArrayField()
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_pullrequest_comment"
+        unique_together = (("pull_request", "external_id"),)
```

Comment:
i'm not sure if this should be `id` or `external_id`

Reply:
I assume you're trying to prevent duplication with this line so for one comment in one pr you don't get two rows. If so then external_id is correct

Reply:
thank you!